tests/functional/fixtures/ext/foo/foo/event/template_event_order.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace foo\foo\event;
+
+/**
+* Event listener
+*/
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class template_event_order implements EventSubscriberInterface
+{
+	static public function getSubscribedEvents()
+	{
+		return array(
+			'core.twig_tokenparser_constructor'	=> 'set_template_event_priority',
+		);
+	}
+
+	public function set_template_event_priority($event)
+	{
+		$template_event_priority_array = $event['template_event_priority_array'];
+		$template_event_priority_array['foo_bar'] = [
```

Comment:
Shouldn't this (and the other file for `foo/foo`) say `foo_foo` instead of `foo_bar` here?

Reply:
Sure.